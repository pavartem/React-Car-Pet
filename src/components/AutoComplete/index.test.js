import React from "react";
import {configure, shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import {AutoComplete} from "./index";
configure({adapter: new Adapter()});

describe('<AutoComplete/>', () => {
    const autoComplete = [{make: 'bmw', model: 'x3'}];
    const setName = jest.fn();
    const toggleAutoComplete = jest.fn();

    const wrapperForSnapshot = shallow(
        <AutoComplete autoComplete={autoComplete}
                      setName={setName}
                      toggleAutoComplete={toggleAutoComplete}
        />);

    it('Should render component', () => {
        expect(wrapperForSnapshot).toMatchSnapshot();
    });
});
