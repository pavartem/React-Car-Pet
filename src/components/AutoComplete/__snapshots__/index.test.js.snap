// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AutoComplete/> Should render component 1`] = `ShallowWrapper {}`;
